examples/acados/pendulum.py

```diff
@@ -108,13 +108,15 @@ def main():
     If pendulum is run as a script, it will perform the optimization using ACADOS and animates it
     """
 
-    ocp = prepare_ocp(biorbd_model_path="pendulum.bioMod", final_time=3, n_shooting=41)
+    ocp = prepare_ocp(biorbd_model_path="pendulum.bioMod", final_time=1, n_shooting=100)
 
     # --- Solve the program --- #
     sol = ocp.solve(solver=Solver.ACADOS)
 
     # --- Show results --- #
     sol.graphs()
+    sol.print()
```

Comment:
Print should be prior to graphs, since graphs is blocking (so it won't appear nicely otherwise)